lib/parsers.js

```diff
@@ -287,15 +287,30 @@ exports.parseColor = function parseColor(val) {
     alpha = 1;
   var parts;
   var res = colorRegEx1.exec(val);
-  // is it #aaa or #ababab
+  // is it #aaa, #ababab, #aaaa, #abababaa
   if (res) {
+    var defaultHex = val.substr(1);
     var hex = val.substr(1);
-    if (hex.length === 3) {
+    if ([3, 4].indexOf(defaultHex.length) > -1) {
```

Comment:
If there are only two values to compare, having the comparison in the code explicitly instead of the much more complicated expression should work better.

I suggest using `hex.length === 3 || hex.length === 4` instead, it reads better.

Reply:
Sure.